lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '*.ts': ["prettier --write ./src/**/*.ts", "tslint --fix", "git add"],
```

Comment:
are you sure that always fixing is good idea?

Reply:
Hm, probably better do manually, will fix